addons/web_tour/static/src/tour_service/tour_service.js

```diff
@@ -70,7 +61,6 @@ function checkTourStepKeyValues(tourStep) {
         position: { type: String, optional: true },
         run: { type: [String, Function], optional: true },
         allowInvisible: { type: Boolean, optional: true },
-        allowDisabled: { type: Boolean, optional: true },
         in_modal: { type: Boolean, optional: true },
         timeout: { type: Number, optional: true },
         consumeEvent: { type: String, optional: true },
```

Comment:
The `consumeEvent` key is removed from the jsdoc but is still used here, is this normal?

Reply:
well seen !